src/content_stream_test.cpp

```diff
@@ -19,6 +19,27 @@
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
 
+#include <memory>
+
+namespace {
+
+// SingleByteReader wraps a dap::Reader to only provide a single byte for each
+// read() call, regardless of the number of bytes actually requested.
+class SingleByteReader : public virtual dap::Reader {
```

Comment:
Why virtual inheritance?

Reply:
Good spot. Copy pasta mistake by the looks of it. Removed.